frontend/lib/parse-capi/index.ts

```diff
@@ -173,7 +179,12 @@ export const extractArticleMeta = (data: {}): CAPIType => {
             .map(block => block.bodyHtml)
             .filter(Boolean)
             .join(''),
-        author: getString(data, 'config.page.author'),
+        author: {
+            byline: getString(data, 'config.page.byline', ''),
+            twitterHandle: leadContributor.properties.twitterHandle,
```

Comment:
If there are no tags where `tagType === 'Contributor'` then `leadContributor` would be `undefined`, if it is looking up `properties` on it'll cause an error.  You could add a ternary to check if it's defined and if not just assign undefined to `twitterHandle`. eg.

`twitterHandle: leadContributor ? leadContributor.properties.twitterHandle : undefined,`

Reply:
thanks!